tests/test_form.py

```diff
@@ -230,6 +230,22 @@ class F(Form):
         assert F(DummyPostData(), test="test").test.data == "processed"
         assert F(DummyPostData({"test": "foo"}), test="test").test.data == "foo"
 
+    def test_errors(self):
+        class F(Form):
+            foo = StringField(validators=[DataRequired()])
+
+            def validate(self):
+                if not super(F, self).validate():
+                    return False
```

Comment:
Since this is a test and `return False` will never happen, just make this a direct call to `super().validate()` rather than an if statement, otherwise you have unreachable code. Also don't need the old-style `super` args.

Reply:
Until #554 is merged there are still python2 tests that may fail if we use the new-style `super()`.